### ISSUE TITLE
Pusher depends on an old version of request that has deprecation notices

### ISSUE THREAD

**Issue by jakegavin** [pusher/pusher-http-node#63]
When I install pusher, I get
```
warning pusher > request > node-uuid@1.4.8: Use uuid module instead
```
Obviously, everything still functions fine but it'd be nice to remove this notice.

[request](https://github.com/request/request) migrated to `uuid` this in [2.79.0](https://github.com/request/request/blob/master/CHANGELOG.md#v2790-20161118) but this library still depends on 2.74.0. 

I tried to fix it so I could open a PR but the timeout tests failed.

**Comment by eivindingebrigtsen** [comment]
+1 on this, with node version 8.2.1, pusher throws error even though node-uuid is installed.

**Comment by leesio** [comment]
Thanks for highlighting this. I'm sorry it took so long for you to receive a response.

I have a PR to update this dependency, I'll be deploying an updated version of the library ASAP.

**Comment by leesio** [comment]
I just released [v2.0.0](https://github.com/pusher/pusher-http-node/releases/tag/v2.0.0) which updates the requests library, and published to npm.

**Comment by leesio** [comment]
I'm going to close this issue now as I think the issue is resolved.
